content/cloud-docs/api-docs/private-registry/providers.mdx

```diff
@@ -222,7 +247,46 @@ curl \
   https://app.terraform.io/api/v2/organizations/my-organization/registry-providers
 ```
 
-### Sample Response (public provider)
+### Sample Response (Private Provider)
+
+```json
+{
+  "data": {
+    "id": "prov-cmEmLstBfjNNA9F3",
+    "type": "registry-providers",
+    "attributes": {
+      "name": "aws",
+      "namespace": "hashicorp",
+      "registry-name": "private",
+      "version-statuses": [],
```

Comment:
Could we please remove the `version-statuses` attribute from this response?

Reply:
done!